### DISCUSSION TITLE
Caps Lock On-Screen Display

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#1424] [Ideas]
hello.

First of all, thank you for developing G-Helper. I'm using it really well.

It's not directly related to G-Helper, but I think it would be nice to have an indication on the screen when typing the Caps-Lock and Num-Lock buttons on the keyboard.

Like the overlay that appears when you press the Fn-Lock button in G-Helper.

I've used other laptops that have the ability to show CapsLock On and CapsLock OFF, and it's very useful.

Thanks

**Comment by seerge** [comment]
@cheonmux hello :) that's generally handy tweak, but I'm afraid it is very generic (i.e. not specific neither to armoury create features nor to asus hardware).

I don't want to overload app with things you can solve somehow else. 

5 seconds of googling has led me to this tool https://cli.jonaskohl.de/ it seem to do exactly what you want :)
